overlord/snapshotstate/snapshotmgr.go

```diff
@@ -371,6 +372,23 @@ func undoRestore(task *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+func doTaskCleanup(task *state.Task, tomb *tomb.Tomb) error {
```

Comment:
Why not just use `cleanupRestore()` directly?

Reply:
I thought I would need to do some other operations before or after it, but it turns out that using `cleaupRestore()` directly is enough. I've removed the wrapper now.